melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/til_sed/SedMapper.java

```diff
@@ -22,32 +23,51 @@
 import no.nav.melosys.eessi.service.sed.helpers.LandkodeMapper;
 import org.springframework.util.StringUtils;
 
-import static no.nav.melosys.eessi.models.sed.Konstanter.DEFAULT_SED_G_VER;
-import static no.nav.melosys.eessi.models.sed.Konstanter.DEFAULT_SED_VER;
+import static no.nav.melosys.eessi.models.SedType.*;
+import static no.nav.melosys.eessi.models.sed.Konstanter.*;
 
 /**
  * Felles mapper-interface for alle typer SED. Mapper NAV-objektet i NAV-SED, som brukes av eux for
  * å plukke ut nødvendig informasjon for en angitt SED.
  */
 public interface SedMapper {
-    default SED mapTilSed(SedDataDto sedData) {
+    default SED mapTilSed(SedDataDto sedData, Boolean erCDM4_3) {
         var sed = new SED();
+        var sedType = getSedType();
 
-        sed.setNav(prefillNav(sedData));
-        sed.setSedType(getSedType().name());
+        sed.setNav(prefillNav(sedData, erCDM4_3));
+        sed.setSedType(sedType.name());
         sed.setSedGVer(DEFAULT_SED_G_VER);
-        sed.setSedVer(DEFAULT_SED_VER);
+
+        if(erCDM4_3){
+            sed.setSedVer(SED_VER_CDM_4_3);
+        } else {
+            sed.setSedVer(DEFAULT_SED_VER);
+        }
+
 
         return sed;
     }
 
     SedType getSedType();
 
-    default Nav prefillNav(SedDataDto sedData) {
+    default Nav prefillNav(SedDataDto sedData, boolean erCDM4_3) {
         var nav = new Nav();
+        var sedType = getSedType();
+
+        if(erCDM4_3){
+            switch (sedType) {
+                case A001, A002, A003, A008, A009, A010 -> {
+                    nav.setArbeidsland(hentArbeidsland(sedData));
+                    nav.setHarfastarbeidssted(sedData.getHarfastarbeidssted() ? "ja" : "nei");
```

Comment:
harFastArbeidssted og getHarFastArbeidssted? 

Er det Rina som har harfastarbeidssted in all lower case?

Reply:
Stemmer, denne her må forbli: nav.setHarfastarbeidssted

Endrer sedData.getHarfastarbeidssted()